crates/core/src/lib.rs

```diff
@@ -213,12 +227,12 @@ impl<T: Send + Sync> EngineBuilder<T> {
         let engine = wasmtime::Engine::new(&config.inner)?;
 
         let mut linker: Linker<T> = Linker::new(&engine);
-        wasmtime_wasi::preview2::command::add_to_linker(&mut linker)?;
+        wasmtime_wasi_http::proxy::add_to_linker(&mut linker)?;
```

Comment:
Note that `wasmtime_wasi_http::proxy::add_to_linker` includes a call to `wasmtime_wasi::preview2::command::add_to_linker`.